rviz_common/include/rviz_common/logging.hpp

```diff
@@ -103,8 +103,8 @@ namespace rviz_common
 {
 
 using LoggingHandler = std::function<
-    void(const std::string & message, const std::string & file_name, size_t line_number)
-    >;
+  void(const std::string & message, const std::string & file_name, size_t line_number)
+>;
```

Comment:
This is the ideal style (imo), which I described here: https://github.com/ros2/rclcpp/pull/445#discussion_r171134238

Reply:
Changing the version of `uncrustify` from master to the latest release made the result less "ideal".

Reply:
That is sad.